Makefile

```diff
@@ -93,8 +94,15 @@ else
 	GO111MODULE=on GOARCH=$(ARCH) go test --test.short ./pkg/... $(FLAGS)
 endif
 
+# Loops through the three most recent releases of Kubernetes.
```

Comment:
Have you looked into separating them into different targets?
As a developer I would like to have option to test only latest version or some particular one. CI should test all supported versions.

Would be nice to have separate targets like:

test-e2e-latest
test-e2e-all
test-e2e-1.17
test-e2e-1.16
test-e2e-1.15